api/src/main/scala/za/co/absa/hyperdrive/ingestor/api/utils/ConfigUtils.scala

```diff
@@ -53,6 +53,16 @@ object ConfigUtils {
     }
   }
 
+  def getBooleanOrNone(key: String, configuration: Configuration): Option[Boolean] = {
```

Comment:
I think this would be straight forward (it's up to you): 
`  
  if(configuration.containsKey(key)) {
      if(configuration.getString(key).isEmpty) {
        Some(true)
      } else {
        Option(configuration.getBoolean(key))
      }
    } else {
      None
    }
`

Reply:
+ Why is empty string true ? Isn't it incorrect value ?